isthmus/src/test/java/io/substrait/isthmus/Substrait2SqlTest.java

```diff
@@ -58,7 +129,10 @@ private void test(String query) throws Exception {
     Rel pojoRel2 =
         SubstraitRelVisitor.convert(RelRoot.of(relnodeRoot, SqlKind.SELECT), EXTENSION_COLLECTION);
```

Comment:
With fix https://github.com/substrait-io/substrait-java/pull/35, does it work to use `ProtoPlanConverter` instead?